.buildkite/pipelines/main/misc/signed_pipeline_test.yml.signature

```diff
@@ -1 +1 @@
-Salted__���J0�Q?���rۀ�g�~�d��ۛŧ�ө��o���Ujʀ���p�)�$�U$���y@gZM}{�m��,۠�K��e�r�
\ No newline at end of file
+Salted__"@�:�"z,͡�Ƣ�t9���"��?�+�'u��"e��:8�Z�kg���>�� ��ȫ.�7U12M~ ������u���
```

Comment:
Where these meant to be changed?

Reply:
Yeah, I blanket re-signed.

Reply:
Yes: the YAML files that these signatures apply to were changed, so the signatures have to change too. Might have been better to not change them, but having all text files end with a single newline seems like a simple and consistent rule that's close to what we were already doing.